src/pkg/manifests/package:pkg:system-repository.p5m

```diff
@@ -72,7 +72,6 @@ file path=var/log/pkg/sysrepo/rewrite.log owner=pkg5srv mode=0644 preserve=true
 license cr_Oracle license=cr_Oracle
 # force a dependency on the six package because we bypass dependency check for it
 depend type=require fmri=library/python/six-35
-depend type=require fmri=package/pkg@$(PKGVERS)
```

Comment:
We should add one dependency on IPS main package from system-repository package though. Or not?

Reply:
```
-depend fmri=package/pkg@0.5.11,5.11-0.151.1.8.1 type=require
...
 depend fmri=pkg:/package/pkg@0.5.11-0.151.1.8.1 type=require
```